blocks/library/gutenvert/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlockType } from '../../api';
+
+registerBlockType( 'promo/gutenberg-footer', {
+	title: __( 'Editor Promo' ),
+
+	icon: 'megaphone',
+
+	category: 'common',
+
+	save() {
+		return (
+			<div>
+				<hr className="wp-block-separator" />
```

Comment:
Do you think we should leave it up to the author to insert this or not? Or maybe use `<footer>` and style that (differently) together with the styles below?

Reply:
`<footer>` sounds good - I'll have to check on styling but I like the idea - thanks!